package.json

```diff
@@ -149,30 +149,29 @@
     ]
   },
   "devDependencies": {
+    "@pmmmwh/react-refresh-webpack-plugin": "0.5.4",
+    "@solana/web3.js": "^1.31.0",
```

Comment:
Bleh, these changes are more far-reaching than I intended, sorry. `npm install --save-dev` for a new `@types` file that the CI was breaking due to not having ended up making all these changes.